src/helpers/db.js

```diff
@@ -0,0 +1,143 @@
+      submission_made INTEGER DEFAULT 0,
+      FOREIGN KEY (feed) REFERENCES jobs(name) ON DELETE CASCADE
+    );
+  `);
+};
+
+/**
+ * Function to get all jobs from the DB
+ * @returns {Object[]} array of jobs in DB
+ * @returns {string} returns[].id The CL job ID
+ * @returns {string} returns[].name The name of the feed
+ * @returns {number} returns[].request_id The ID of the last request made to *                   the CL node
+ * @returns {number} returns[].last_reported_round The latest submitted round
+ * @returns {number} returns[].last_request_sent The timestamp when the last CL *                   job request was sent
+ * @returns {number} returns[].last_submission_time The timestamp when the last *                   on-chain price submission was made
+ * @returns {number} returns[].last_result The last on-chain price
+ * @returns {number} returns[].last_received_request_id The ID of the last *                   request whose response was received from the CL node
+ */
+export const getAllJobs = async () => {
+  return new Promise((resolve, reject) => {
+    db.all("SELECT * FROM jobs", (err, rows) => {
+      if (err) {
+        console.log("DB ERROR:", err);
+        reject([]);
+      } else {
+        resolve(rows);
+      }
+    });
+  });
+};
+
+/**
+ * Function to add a job to the DB
+ * @param {string} id id of the job
+ * @param {string} name name of the feed
+ */
+export const createJob = async (id, name) => {
+  try {
+    await db.run("INSERT INTO jobs (id, name) VALUES (?, ?)", [id, name]);
+    await db.run("INSERT INTO rounds (feed) VALUES (?)", [name]);
+  } catch (err) {
+    console.log("DB ERROR:", err);
+  }
+};
+
+/**
+ * Function to delete a job from the DB
+ * @param {string} id id of the job to delete
+ */
+export const deleteJob = async (id) => {
+  try {
+    await db.run("DELETE from jobs where id = '" + id + "';");
+  } catch (err) {
+    console.log("DB ERROR:", err);
+  }
+};
+
+/**
+ * Function to make a query to a table
+ * @param {string} table table name
+ * @param {string[]} fields fields to obtain
+ * @param {string} name feed name to query
+ * @returns {Object} an object containing the state of the job for the given *                   feed and fields
+ */
+export const queryTable = async (table, fields, name) => {
+  let keyName = table === "jobs" ? "name" : "feed";
+  return new Promise((resolve, reject) => {
+    db.get(
+      "SELECT " + fields.join(", ") + " from " + table + " where " +
+      keyName + " = '" + name + "';",
+      (err, rows) => {
+        if (err) {
+          console.log("DB ERROR:", err);
+          reject({});
+        } else {
+          resolve(rows);
+        }
+      }
+    );
+  });
+};
+
+/**
+ * Function to make an update to a table
+ * @param {string} table table name
+ * @param {Object} values values to update in a JSON object. The properties *                        would be the column names and the values would be the *                        values to set
```

Comment:
```suggestion
 * @param {Object} values values to update in a JSON object. The properties 
 *                        would be the column names and the values would be the 
 *                        values to set
```

Reply:
Fixed in https://github.com/jacquesvcritien/agoric-cl-middleware/commit/9f027c8cb34abf5790b78708b269ee97944c051e